modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+Adding `OR REPLACE` to the command will result in any existing database alias being deleted and a new one created.
+`CREATE OR REPLACE ALIAS` will fail if there is an existing database with the same name.
+
+[NOTE]
+====
+The `IF NOT EXISTS` and `OR REPLACE` parts of this command cannot be used together.
+====
+
+[NOTE]
+====
+Database alias names are subject to the rules specified in the xref:administration/alias-management-escaping[Alias names and escaping] section.
+====
+
+[[database-management-create-local-database-alias]]
+=== Create local database aliases
+
+A local database alias targets a database within the same DBMS.
+
+==== Create aliases for local databases
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2021`
+----
+
+
+When you create a local database alias, it shows up in the `aliases` column provided by the command `SHOW DATABASES` and in the `SHOW ALIASES FOR DATABASE` command.
+
+
+.Query
+[source, cypher]
+----
+SHOW DATABASE `northwind`
+----
+
+.Result
+[role="queryresult"]
+----
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                   | type       | aliases       | access       | address          | role      | writer | requestedStatus | currentStatus | statusMessage | default | home  | constituents |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "northwind-graph-2021" | "standard" | ["northwind"] | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []           |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+.Query
+[source, cypher]
+----
+SHOW ALIAS `northwind` FOR DATABASE
+----
+
+.Result
+[role="queryresult]
+----
++---------------------------------------------------------------------------+
+| name        | composite | database               | location | url  | user |
++---------------------------------------------------------------------------+
+| "northwind" | NULL      | "northwind-graph-2021" | "local"  | NULL | NULL |
++---------------------------------------------------------------------------+
+----
+
+==== Set properties for local database aliases
+
+Local database aliases can also be given properties.
+These properties can then be used in queries with the xref::database-administration/composite-databases/querying-composite-databases.adoc#composite-databases-queries-built-in-functions[`graph.propertiesByName()` function].
```

Comment:
previously this linked to the documentation for the function itself, not it links to the composite database querying section, is that what we want?

Reply:
No, it is probably better to link to the main page for the function.